README.md

```diff
@@ -201,6 +292,49 @@ File 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267
 	Observed filenames: 131f95c51cc819465fa1797f6ccacf9d494aaaff46fa3eac73ae63ffbdfd8267,eicar.com.txt,eicar.txt
 ```
 
+```bash
+$ polyswarm_api -o /tmp/test.txt search search metadata '{ "query": { "exists": {"field": "lief.libraries"}}}'
```

Comment:
should there be 2 "search" words in a row?

Reply:
I don't know how I put this in there. Will fix.